src/components/Popups/PopupCursor.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import PopupBase from './PopupBase';
+import { Options } from './Popper';
+import './PopupCursor.scss';
+
+export type Props = {
+    isActive: boolean;
+};
+
+const options: Partial<Options> = {
+    modifiers: [
+        {
+            name: 'offset',
+            options: {
+                offset: [0, 20],
+            },
+        },
+    ],
+    placement: 'right',
+};
+
+export default class PopupCursor extends React.PureComponent<Props> {
```

Comment:
It seems like this could be a function component with a couple of usages of `React.useRef`.